src/Builder/FrontendDialectTransformer.hpp

```diff
@@ -55,6 +55,16 @@ struct ImportOptions {
   //   - (arg0: tensor<3x4x5xf32>, arg1: tensor<10x5xf32>)
   //
   std::string shapeInformation = "";
+  // Custom onnx.dim_params attributes for the graph inputs for specifying
+  // relationship among their ranks.
```

Comment:
`their ranks` looks confusing. To be precise, it would be `their dynamic dimensions`, given that `dim` in `dim_params` means dimension.

Reply:
Fixed. Thanks!